examples/resources/saml-connector.yaml

```diff
@@ -8,17 +8,18 @@ metadata:
   # the name of the connector
   name: new_saml_connector
 spec:
-  # display allows to set the caption of the "login" button
-  # in the Web interface
-  display: "Login with SSO"
-  # acs is the Assertion Consumer Service URL. This should be the address of
-  # the Teleport proxy that your identity provider will communicate with.
-  acs: https://teleport-proxy.example.com:3080/v1/webapi/saml/acs
+  # connector display name that will be appended to the title of "Login with"
+  # button on the cluster login screen so it will say "Login with Okta"
+  display: Okta
```

Comment:
Weirdly we already fixed this one but none of the others...

Reply:
I've updated the rest now.